### ISSUE TITLE
final review of week 4 blog entries

### ISSUE THREAD

**Issue by nicholasjhorton** [hglanz/DS-Blog-Ex#35]
Jo
- [x]  Monday 6/17 infer (Jo)
- [x]  Tuesday 6/18 SQL 101 (Nick)
- [x]  Wednesday 6/19 mosaic (Nick)
- [x]  Thursday 6/20 PeerJ (Nick) [was guest entry: programming tips (Kelly)]
- [x]  Friday 6/21 pair programming (Nick)

Hunter
- [x]  Monday 6/17 infer (Jo)
- [x]  Tuesday 6/18 SQL 101 (Nick)
- [x]  Wednesday 6/19 mosaic (Nick)
- [x]  Thursday 6/20 PeerJ (Nick) [was guest entry: programming tips (Kelly)]
- [x]  Friday 6/21 pair programming (Nick)

Nick

- [x]  Monday 6/17 infer (Jo)
- [x]  Tuesday 6/18 SQL 101 (Nick)
- [x]  Wednesday 6/19 mosaic (Nick)
- [ ]  Thursday 6/20 PeerJ (Nick) [was guest entry: programming tips (Kelly)]
- [x]  Friday 6/21 pair programming (Nick)

**Comment by nicholasjhorton** [comment]
there's a chunk of remaining items to close out for the week 4 milestone (sorry that I left so much for today)
<img width="1010" alt="Screen Shot 2019-06-14 at 5 14 14 PM" src="https://user-images.githubusercontent.com/530542/59538489-f1ed2f00-8ec7-11e9-8720-fe85a086f70c.png">

**Comment by hglanz** [comment]
So I'm pretty late in getting to things and I apologize. I've approved all of the entries. They look great! One (broad) question:

- There are a handful of packages that work with integrating R and SQL (or at least SQL syntax). Among the ones I talk about in class is sqldf. Are the ones that @nicholasjhorton included in the SQL entry ubiquitous or more mainstream, or is there a reason not to mention all (or close to all) of the packages out there for this sort of thing?

Great work you two!

**Comment by hardin47** [comment]
my two cents is that we should include (in "learn more" perhaps) all the R packages that are in rotation among R/SQL users.

**Comment by nicholasjhorton** [comment]
Regarding `sqldf`, I've always thought of this as a mechanism to interact with R dataframes as if they were a database.  Is that a better mechanism for an existing database?  I'd be interested in hearing more (or potentially adding an activity that you've used with that package.
 
I took a more general added an explicit shoutout to the CRAN Task View on Databases with R which includes a mention of `sqldf`.  I also noted the planned shift from RMySQL to RMariaDB.

Fixed via https://github.com/hglanz/DS-Blog-Ex/commit/5fd1b7b1d92706dee05af5dfee0c339b614bbe0f and https://github.com/hglanz/DS-Blog-Ex/commit/fbe03564b4835e618ff0b3a1b6d5dc612f28f8e1
